mail/mailer.go

```diff
@@ -212,32 +216,40 @@ func (m *MailerImpl) SendMail(to []string, subject, msg string) error {
 	if m.client == nil {
 		return errors.New("call Connect before SendMail")
 	}
+	m.stats.Inc("SendMail.Attempts", 1)
 	body, err := m.generateMessage(to, subject, msg)
 	if err != nil {
+		m.stats.Inc("SendMail.Errors", 1)
```

Comment:
Usually when I find myself repeating a stats or log line on every error, I find it's better to factor out a function that returns error, call it, and stat the error once in the calling function.